### PR TITLE
update readme to new format

### PR DESCRIPTION
**Pull request by StoicJester**
@jrit @Tathanen tried to match the new format from vokal-api

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,27 +14,75 @@ When creating your main angular module, include `vokal.majora` in the list of in
...
+### Methods
+
+#### <a name="method-toggleBits"></a>`toggleBits( number, bitsArray, newValue )`
+
+Given an input `number`, update the bits matched by `bitsArray` to be `newValue`
+
+##### Arguments
+
+1. `number`    | *Number* | the number to be mutated
+
```
Comment (Tathanen) [inline README.md]: I'd pull out these blank lines between arguments, just keep the section a bit more condensed.

File: README.md [hunk 2]
```diff
@@ -12,27 +14,75 @@ When creating your main angular module, include `vokal.majora` in the list of in
...
+2. `bitsArray` | *Number* | selects the bits to mutate in number
+
+3. `newValue`  | *Bool*   | new value of the bits selected by bitsArray
+
+##### Response
+
+Returns a *Number*, which is the decimal representation of the `number` argument with the selected bits modified
```
Comment (Tathanen) [inline README.md]: Header should be "Returns", and follow similar format as arguments.  That is,

```
*Number* | the decimal representation of the `number` argument with the selected bits modified
```

File: README.md [hunk 3]
```diff
@@ -12,27 +14,75 @@ When creating your main angular module, include `vokal.majora` in the list of in
...
+* * *
+
+#### <a name="method-testOptimistic"></a>`testOptimistic( number, bitsArray )`
+
+Given an input `number`, check if any of the bits in `bitsArray` are in that number
+
+##### Arguments
+
+1. `number`    | *Number* | the number to be tested
+
```
Comment (Tathanen) [inline README.md]: Blank lines

File: README.md [hunk 4]
```diff
@@ -12,27 +14,75 @@ When creating your main angular module, include `vokal.majora` in the list of in
...
+#### <a name="method-testPessimistic"></a>`testPessimistic( number, bitsArray )`
+
+Just like testOptimistic, but tests from bitsArray must be true for the result to be true.  
+All numbers must have some true bit overlap to be true, does not require complete overlap.
+
+##### Arguments
+
+1. `number`    | *Number* | the number to be tested
+
```
Comment (Tathanen) [inline README.md]: Blank lines

File: README.md [hunk 5]
```diff
@@ -12,27 +14,75 @@ When creating your main angular module, include `vokal.majora` in the list of in
...
+2. `bitsArray` | *Number* | the bits to test in number
+
+##### Response
+
+Returns a *Bool*, which indicates whether all of the bits in `bitsArray` are active in `number`
```
Comment (Tathanen) [inline README.md]: "Returns", and

```
*Boolean* | indicates whether all of the bits in `bitsArray` are active in `number`
```

File: README.md [hunk 6]
```diff
@@ -12,27 +14,75 @@ When creating your main angular module, include `vokal.majora` in the list of in
...
+
+##### Arguments
+
+1. `number`    | *Number* | the number to be tested
+
+2. `bitsArray` | *Number* | the bits to test in number
+
+##### Response
+
+Returns a *Bool*, which indicates whether any of the bits in `bitsArray` are active in `number`
```
Comment (Tathanen) [inline README.md]: "Returns", and 

```
*Boolean* | indicates whether any of the bits in `bitsArray` are active in `number`
```

**Comment by StoicJester** [comment]
changes made

**Comment by Tathanen** [comment]
LGTM

**Pull request merged**